collect_app/src/main/java/org/odk/collect/android/database/helpers/InstancesDatabaseHelper.java

```diff
@@ -156,6 +161,27 @@ private void upgradeToVersion4(SQLiteDatabase db) {
         }
     }
 
+    private void downgradeToVersion5(SQLiteDatabase db) {
+        Cursor cursor = db.query(INSTANCES_TABLE_NAME, null, null, null, null, null, null);
+        if (cursor != null && cursor.moveToFirst()) {
+            List<Instance> instances = new InstancesDao().getInstancesFromCursor(cursor);
+            for (Instance instance : instances) {
+                if (instance.getInstanceFilePath().startsWith("../")) {
+                    String where = INSTANCE_FILE_PATH + "=?";
+                    String[] whereArgs = {instance.getInstanceFilePath()};
```

Comment:
Can we use instance ID and index it? I can't believe there isn't already an index for it.

Reply:
Yes, done.